percona/pmm/pmm.go

```diff
@@ -267,6 +277,20 @@ func sidecarContainerV3(pgc *v2.PerconaPGCluster) corev1.Container {
 	if pgc.Spec.PMM.CustomClusterName != "" {
 		clusterName = pgc.Spec.PMM.CustomClusterName
 	}
+	volumeMounts := []corev1.VolumeMount{
+		{
+			Name:      "cert-volume",
+			MountPath: "/pgconf/tls",
+			ReadOnly:  true,
+		},
+	}
+	if pgc.CompareVersion("2.7.0") >= 0 {
```

Comment:
We can skip this part with the compare version for PMM3 given we are introducing it in the 2.7 as a whole.

Reply:
https://github.com/percona/percona-postgresql-operator/pull/1113/commits/9816674fffaf0bf2fc62a26f77ab73c1301770b3